### ISSUE TITLE
Update benchmarks?

### ISSUE THREAD

**Issue by iangow** [bwlewis/duckdb_and_r#2]
https://github.com/bwlewis/duckdb_and_r/blob/d4de43f682719fae54274887d7d01993f53c3e4c/tpch/tpch.rmd#L206-L211

I see much faster performance for the DuckDB options (see [here](https://github.com/iangow/notes/blob/main/tpch-duckdb.pdf)). Not clear whether this is due to differences in architecture (I'm using a 2020 Mac mini M1 with 16GB of RAM and the data on an external drive) or due to developments in DuckDB.

**Comment by bwlewis** [comment]
Indeed, I wrote this a long time ago and DuckDB has come a long way (but so has pola.rs). It will be interesting to re-run and expand.
